### PR TITLE
Desktop & Mobile-view

### PR DESCRIPTION
**Pull request by Devsalmasaeed**
Added html and css files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        <div class="header">
+        <div class="overlap-group">
+        <div class="blurry"></div>
+        <div class="circle-background"><div class="ellipse"></div></div>
+        <p class="travel-around-the">
+        <span class="text-wrapper">Travel</span> <span class="span"> around the world</span>
+        </p>
+        <img class="image" src="img/Image (4).png" />
+        </div>
+    </section>
```
Comment (devalibello) [inline index.html]: Comment on Indentation: The section was correctly implemented but the children tags within the section tag should be indented. E.g 
<section>
    <div>Text 1</div>
    <div>Text 1</div>
</section>

Go through your ...

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+    <section class="content">
+        <div class="content">
+            <p>Find your best destination</p>
+    
+        <span>We have more than 1000+ destinations you can choose.</span>
```
Comment (devalibello) [inline index.html]: Remove all white spaces within your HTML file

**Comment by Xfarouq** [comment]
## Nice Work!💯
- Code looks good
- Correctly Applied the use of Semantics
- Good indentation too

# Code Defect
- Put in your mobile screen size code into your Desktop screen code and your good to go✅

File: styles.css
Comment (Xfarouq) [inline styles.css:1]: ## Nice Work!💯
- Code looks good
- Correctly Applied the use of Semantics
- Good indentation too

# Code Defect
- Put in your mobile screen size code into your Desktop screen code and your good to go✅

**Review by Xfarouq — CHANGES_REQUESTED** [review]

**Pull request merged**